ui/package.json

```diff
@@ -17,8 +17,10 @@
   },
   "scripts": {
     "start": "yarn install && yarn generate && cross-env TS_NODE_PROJECT=\"webpack.tsconfig.json\" && yarn run build:vendor && yarn run start:dev",
-    "start:docker": "yarn generate && yarn build:vendor && yarn run start:dev",
+    "start:cloud": "yarn install && yarn generate && cross-env TS_NODE_PROJECT=\"webpack.tsconfig.json\" && yarn run build:vendor && yarn run start:cloud-dev",
     "start:dev": "webpack-dev-server --config ./webpack.dev.ts",
+    "start:cloud-dev": "CLOUD=true CLOUD_LOGOUT_URL=http://localhost:8080/api/v2/signout CLOUD_URL=http://localhost:9999 webpack-dev-server --config ./webpack.dev.ts",
```

Comment:
`CLOUD=true` is no longer required. Should we also be more consistent with the cross-env usage? Seems weird that `start:cloud` is using it to be windows compliant, but `start:cloud-dev` isn't